### PR TITLE
Feature/additional principles and introduction

### PR DESCRIPTION
**Pull request by RobTobias123**
Adding principles page to guidelines and introduction (was about)  outside the containers

### CODE REVIEW

File: src/content/docs/introduction.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+title: Nucleus design system
+---
+
+## What is Nucleus?
+
+<div class="purpose">
+Nucleus is the Design System of British Gas. Its purpose is to help provide cohesive and coherent web experiences.
+</div>
+
+### Our vision 🌟
```
Comment (andij) [inline src/content/docs/introduction.mdx:11]: Should we remove the emoji from these headings?

I do like them. But to me it feels somewhat dated.

File: src/content/docs/introduction.mdx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### Our mission 🚀
+
+Nucleus is an inclusive, community driven design system. It brings together our brand personality and world class experiences.
+
+### Our purpose 🧩
+
+Empowering teams to collaboratively build reusable, coherent and accessible experiences that delight customers by using our aligned principles and processes.
+
+### What is a design system?
```
Comment (andij) [inline src/content/docs/introduction.mdx:23]: This I think should be an h2.

```suggestion
## What is a design system?
```

File: src/content/docs/introduction.mdx (hunk 1 above)
Comment (RobTobias123) [reply src/content/docs/introduction.mdx:11]: Yeah I was in 2 minds about it yesterday - I'll remove for a more formal look.

File: src/content/docs/guidelines/principles.mdx [hunk 3]
```diff
@@ -0,0 +1,137 @@
+---
+title: Principles
```
Comment (andij) [inline src/content/docs/guidelines/principles.mdx:2]: Reduce the overloaded sidebar.

```suggestion
title: Principles
tableOfContents:
  maxHeadingLevel: 2
```

File: src/content/docs/guidelines/principles.mdx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+At our heart, we’re a blue and white brand, we use the white space of a page in a considered way so our customers are never overwhelmed.
+
+Being clean and fresh helps us show our content simply, so our customers can get what they need - and get on with their day.
+
+### Warm and friendly
+
+We’re a brand with clarity, but we’re never clinical. Being warm and friendly helps us make a genuine connection with our customers.
+
+## What's the best use for the principles?
```
Comment (andij) [inline src/content/docs/guidelines/principles.mdx:131]:
```suggestion
## How to use these principles
```

**Review by andij — APPROVED** [review]

**Pull request merged**